src/Miso/Canvas.hs

```diff
@@ -166,22 +166,25 @@ import           Miso.Html.Types
 import           Miso.Style (Color, renderColor)
 import           Miso.String (MisoString)
 -----------------------------------------------------------------------------
--- | Element for drawing on a <canvas>, includes a @Canvas@ DSL.
+-- | Element for drawing on a [\<canvas\>](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/canvas).
 -- This function abstracts over the context and interpret callback,
 -- including dimension ("2d" or "3d") canvas.
+--
+-- Unlike @Miso.HTML.Element.canvas@, which just creates an empty canvas element,
+-- this functions (TODO does what?)
```

Comment:
@dmjio could you please help me finish this TODO + the one below? I'm having difficulties understanding how exactly this is to be used / what the JSM JSVal argument represents and I assume others will run into this as well.